app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -38,6 +38,9 @@ def refresh
             post_refresh_ems_cleanup(ems, targets)
           end
 
+          # We need to reload the ems, since it can try to autosave a models that did not pass a validation, and the
+          # ems status with the error would not save.
+          ems.reload
```

Comment:
@blomquisg yeah I was thinking about that, that would be probably for next PR? So we can make this small possible backport?